meson.build

```diff
@@ -56,6 +56,15 @@ dxvk_nvapi_version = vcs_tag(
   input:  'version.h.in',
   output: 'version.h')
 
+conf_data = configuration_data()
+conf_data.set('BUILD_COMPILER', cpp.get_argument_syntax())
+conf_data.set('BUILD_TARGET', dxvk_cpu_family)
+conf_data.set('BUILD_TYPE', get_option('buildtype'))
```

Comment:
This correctly resolves to `debug` when `--buildtype` is not specified when running `meson setup`. 
(This isn't a good idea anyway)

Reply:
> (This isn't a good idea though)

I don't quite understand, why?

Reply:
What I meant with this side note is, using `--buildtype debug` (or omitting `--buildtype`) isn't a good idea because it take ages to compile/link, if it doesn't run out of memory. For debugging, `debugoptimized` should be used.

Reply:
As far as I know only tests have this issue, the main DLL should still be buildable in debug configuration…

… or is that not the case anymore? :thinking:

Reply:
yeah correct, only the tests have this issue.

Reply:
Okay, in that case I wouldn't consider it to be something that needs to be addressed here. Maybe we could have Meson log some warning if tests are enabled and buildtype is debug but I'd consider it out of scope of this PR.

Reply:
Yes, I only added this note to let me and you know that `get_option('buildtype')` works when `--buildtype` isn't explicitly specified. Using `get_option('foo')` bails out if `foo` isn't a valid meson option.